### ISSUE TITLE
ocenaudio fails to build on NixOS 23.11

### ISSUE THREAD

**Issue by boustanihani** [NixOS/nixpkgs#297504]
`ocenaudio` fails to build on `NixOS 23.11`

```
error: hash mismatch in fixed-output derivation '/nix/store/zsb3s3qqs1rrr2bzwzvcics6gj5k2llh-ocenaudio_debian9_64.deb?version=3.13.4.drv':
         specified: sha256-vE+xwwkBXIksy+6oygLDsrT8mFfHYIGcb6+8KMZe0no=
            got:    sha256-9eDDllf6D2hAhjztPhi1MrUky7zQfiE4uLAbfRDx+7s=
error: 1 dependencies of derivation '/nix/store/kv6pqb662kvbbf38cfjpzggwxis849s3-ocenaudio-3.13.4.drv' failed to build
error: 1 dependencies of derivation '/nix/store/9xk1w5wf6ll80incr13q5hc0i0sbjpdn-system-path.drv' failed to build
error: 1 dependencies of derivation '/nix/store/aly864jnq0j5w3kp2qxafgzxxslz7rq8-nixos-system-i7-4770T-23.11.5474.fa9f817df522.drv' failed to build
```

![image](https://github.com/NixOS/nixpkgs/assets/4227548/5c2715f4-2496-4ba3-a3eb-477fbc711f09)

**Comment by chewblacka** [comment]
Should be fixed by backporting this: https://github.com/NixOS/nixpkgs/pull/287500 which I've just initiated.

**Comment by boustanihani** [comment]
Nice, then I'll close this issue when building succeeds :)

**Comment by chewblacka** [comment]
> Nice, then I'll close this issue when building succeeds :)

Sorry it won't let me backport the PR to 23.11, due to a change in syntax for the command `substituteInPlace`.

**Comment by boustanihani** [comment]
So it would be fixed with NixOS 24.05 right ?

**Comment by chewblacka** [comment]
> So it would be fixed with NixOS 24.05 right ?

Yes. It's currently working in unstable.

**Comment by boustanihani** [comment]
Okay, thanks, I'll close this bug :)
